repos/fsharp.proj

```diff
@@ -11,6 +11,7 @@
     <BuildCommand>$(ProjectDirectory)build$(ShellExtension) $(BuildCommandArgs)</BuildCommand>
 
     <PackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</PackagesOutput>
+    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
```

Comment:
Any history here? How are these packages getting into the source-build flow?

Reply:
When `OutputPlacementRepoApiImplemented` is false and `PackagesOutput` is set, the `CopyPackage` task copies the built files.  Packages weren't being copied to the output for fsharp.